eos-tech-support/eos-user-stress-test

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
+
+set -euo pipefail
+
+DATA_FILES=~/eos-qa-test-files/user-tests
+SLEEP_APP_LAUNCH=30
+SLEEP_URL_LAUNCH=10
+
+command=""
+if [ $# -gt 0 ]; then
+        command=$1
+fi
+
+check_dep_flatpak() {
+    # relies on set -e
+    flatpak info $1 > /dev/null
+}
+
+check_deps_common() {
+    if [ ! -e $DATA_FILES ]; then
+        echo >&2 "missing data files at $DATA_FILES"
+        exit 1
```

Comment:
Please try to wget the files and put them in the correct place, and only exit if the files are unavailable (e.g., if no network connection).

Reply:
Fixed in the new version. I'm just downloading the master zip file for the git repo since it seems like a relatively stable URL. In the worst case, ensuring the test files already exist skips the download phase.